site/content/contribute/more-info/server/feature-flags.md

```diff
@@ -158,37 +177,22 @@ When the feature is rolled out to customers, logs will show if there are crashes
 
 ## Self-hosted releases
 
-For a feature-flagged feature to be included in a self-hosted release, the feature flag should be removed.
-
-Feature flags are generally off by default and self-hosted releases do not contact the management system. Therefore feature flags that are not ready for a self-hosted release will be automatically disabled for all self-hosted releases.
-
-Optionally the feature flag can be set to ``true`` in code if we are not yet ready to fully remove the feature flag. Some [examples are here](https://github.com/mattermost/mattermost-server/blob/master/model/feature_flags.go#L75).
+For a feature-flagged feature to be included in a self-hosted release, the feature flag should be either removed or set to ``true`` in code. The feature flag shouldn't be set to ``true`` in code if we're not yet ready to fully remove the feature flag. Some [examples are here](https://github.com/mattermost/mattermost-server/blob/master/model/feature_flags.go#L75).
```

Comment:
> The feature flag shouldn't be set to ``true`` in code if we're not yet ready to fully remove the feature flag.

I'm not understanding because in the previous statement we said it was perfectly OK to release a feature flag in self-hosted with a `true` value.

Reply:
Clarified the text 👍